package.json

```diff
@@ -31,7 +31,7 @@
     "dependencies": {
         "comb": "~1.0.0",
         "comb-proxy": "~1.0.0",
-        "commander": ">=0.5.1",
+        "commander": "~2.9.0",
```

Comment:
talked in person and got it figured out.  One thing we discussed was potentially doing `<= 2.9`.

I think we are good as is though.